homeassistant/components/sonarr/sensor.py

```diff
@@ -245,7 +245,7 @@ def extra_state_attributes(self) -> dict[str, str] | None:
                 identifier = f"S{item.seasonNumber:02d}E{item.episodeNumber:02d}"
 
                 name = f"{item.series.title} {identifier}"
-                attrs[name] = item.airDate
+                attrs[name] = item.airDate.strftime("%Y-%m-%d")
```

Comment:
```suggestion
                attrs[name] = item.airDate.isoformat()
```

Reply:
This would be breaking. maybe not for this PR.

Reply:
This PR is already a breaking change because it deprecates V2 API, so it would be easier to fit this in now.

Reply:
updated but noticed some issues with tzinfo where the aiopyarr could probably help by setting UTC for fields that are sent in UTC from API